src/grandorgue/model/pipe-config/GOPipeConfig.h

```diff
@@ -80,6 +81,7 @@ class GOPipeConfig {
   }
 
   void SetPitchMember(float cents, float &member);
+  void SetToneBalanceMember(int value, int &member);
```

Comment:
This function is not necessary because we have only one ToneBalance member: `m_ToneBalanceValue`